lib/AFS/CellCC/Util.pm

```diff
@@ -401,4 +401,18 @@ calc_checksum($$$$$$) {
     return "$algo:$checksum";
 }
 
+# Get the value of the field provided as an argument for every job and return
+# these values as an array.
+sub
+get_jobs_info($@) {
```

Comment:
eh... I don't think this is needed. This can be replaced by a pretty simple map call, like: `map { $_->{jobid} } @jobs;` (this is really a comment on the whole commit, but github doesn't seem to let me comment on the commit itself?)